bokehjs/src/coffee/embed.ts

```diff
@@ -71,7 +71,7 @@ function _update_comms_callback(target: string, doc: Document, comm: Comm): void
 }
 
 function _init_comms(target: string, doc: Document): void {
-  if (Jupyter != null && Jupyter.notebook.kernel != null) {
+  if (typeof Jupyter !== 'undefined' && Jupyter != null && Jupyter.notebook.kernel != null) {
```

Comment:
You could remove `Jupyter != null`, because the `undefined` case of this check is already considered by the new clause, and `null` case shouldn't happen, nor is permitted by the declaration of `Jupyter` (at the beginning of the file).